src/features/student/views/StudentToolsTracesView/components/StudentTraceDetails/StudentTraceDetails.vue

```diff
@@ -13,23 +17,16 @@ interface StudentDetailedTraceInformationProps {
 
 const props = defineProps<StudentDetailedTraceInformationProps>()
 
-const { t, locale } = useI18n()
+const { t } = useI18n()
 
-function formatDate (date: string) {
-  const currentLocale: AvLocale = locale.value as AvLocale
-  const createdAtDate = new Date(date)
-  return format(createdAtDate, 'dd MMMM yyyy à HH:mm', {
-    locale: currentLocale === 'fr' ? fr : enUS
-  })
+const attachment = computed(() => props.trace.attachment)
+
+const { attachmentFile, uploadDate } = useTraceAttachmentFile(attachment)
+
+const traceFileUploadLabel = computed<string>(() => {
```

Comment:
```suggestion
const traceFileUploadLabel = computed(() => {
```
C'est forcément un string, pas sûr de l'intérêt du typage explicite ici